File: ARCADE/The Core/Level 2_7: willYou?.cs
    using System;
     
    public class Test
    {
    	public static bool willYou(bool young, bool beautiful, bool loved){
            if (young == true && beautiful == true && loved == false) return true;
            if ((young == false || beautiful == false) && loved == true) return true;
            return false;
        }
    	public static void Main()
    	{
    		bool young = true, beautiful = true, loved = true;
    		Console.WriteLine(willYou(young, beautiful, loved));
    		bool young1 = true, beautiful1 = false, loved1 = true;
    		Console.Write(willYou(young1, beautiful1, loved1));
    	}
    }

File: ARCADE/The Core/Level 2_7: willYou?.pas
program willYou;

function willYou(young: boolean; beautiful: boolean; loved: boolean): boolean;
begin
    if((young = true) and (beautiful = true) and (loved = false)) then willYou := true 
    else if(((young = false) or (beautiful = false)) and (loved = true)) then willYou := true
         else willYou := false;
end;

var young, beautiful, loved : boolean;
begin
	young := true; beautiful := true; loved := true;
	writeln(willYou(young, beautiful, loved));
	young := true; beautiful := false; loved := true;
	writeln(willYou(young, beautiful, loved));
end.
